src/common/transformations/src/transformations/symbolic_transformations/reshape_optimizations.cpp

```diff
@@ -0,0 +1,54 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/symbolic_transformations/reshape_optimizations.hpp"
+
+#include <openvino/core/dimension_tracker.hpp>
```

Comment:
please use " instead of < for ov and transformations headers in this and other files in the PR